<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileListScreen</class>
 <widget class="QWidget" name="ProfileListScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>192</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="profileScrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaContent">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>519</width>
        <height>174</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="buttonLayout">
       <item alignment="Qt::AlignmentFlag::AlignTop">
        <widget class="QWidget" name="contentWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="mealButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Meal</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exerciseButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Exercise</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exerciseButton</sender>
   <signal>clicked()</signal>
   <receiver>ProfileListScreen</receiver>
   <slot>navigateToExerciseScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mealButton</sender>
   <signal>clicked()</signal>
   <receiver>ProfileListScreen</receiver>
   <slot>navigateToMealScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ProfileListScreen</sender>
   <signal>addClicked()</signal>
   <receiver>ProfileListScreen</receiver>
   <slot>createProfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>addClicked()</signal>
  <slot>navigateToActionsScreen(ProfileData*)</slot>
  <slot>navigateToExerciseScreen()</slot>
  <slot>navigateToMealScreen()</slot>
  <slot>createProfile()</slot>
 </slots>
</ui>
